### ISSUE TITLE
expo install skips scoped packages

### ISSUE THREAD

**Issue by semmerich-heap** [expo/expo#17918]
### Summary

I am working on developing a plugin that is scoped to our company and when I run expo install the package is skipped for configuration because it has an @ symbol. From what I can tell the CLI splits package names on @ expecting a version number but if the package is scoped it just returns an empty value.  

### Environment

  expo-env-info 1.0.3 environment info:
    System:
      OS: macOS 12.4
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 14.17.4 - /usr/local/bin/node
      Yarn: 1.22.5 - /usr/local/bin/yarn
      npm: 6.14.14 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.9.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
      Android SDK:
        API Levels: 23, 24, 25, 26, 28, 29
        Build Tools: 23.0.1, 25.0.2, 26.0.0, 28.0.3
        System Images: a... | Intel x86 Atom_64, a...gle_apis | Google APIs Intel x86 Atom_64 ..., a... | Intel x86 Atom_64, a...gle_apis | Google APIs Intel x86 Atom_64 ..., a...google_apis | Google APIs Intel x86 Atom Sys...
    IDEs:
      Android Studio: 4.0 AI-193.6911.18.40.6514223
      Xcode: 13.4/13F17a - /usr/bin/xcodebuild
    npmGlobalPackages:
      expo-cli: 5.4.9
    Expo Workflow: managed

### Reproducible demo

expo install @myscope/packagename <- I don't have an example package to use here since ours is not deployed.

**Comment by semmerich-heap** [comment]
Ok so I pushed a package to npm  called `@sde-heap/example` it is a config plugin that adds a line to android strings.xml. If you run `expo install @sde-heap/example` it will install the package but does not add it to the plugins array or recognize that it is a plugin. I did notice if you change expo cli's installAsync.ts line 147 to check if the package starts with an `@` symbol instead of just splitting on `@` it will install and work correctly. I am unsure of any other side effects of that change, but I can raise a PR if that change is OK.
